File: ignored-test/python-unittest/MyFluentWrapper_CallSiteShouldWorkForAsyncMethodsWithReturnValue.py
import unittest

class TestMyFluentWrapper(unittest.TestCase):
    @unittest.skip("")
    def test_call_site_should_work_for_async_methods_with_return_value(self):
        call_site = get_async_call_site()
        self.assertEqual("NLog.UnitTests.LayoutRenderers.CallSiteTests.GetAsyncCallSite", call_site)

File: ignored-test/csharp-xunit/MyFluentWrapper_CallSiteShouldWorkForAsyncMethodsWithReturnValue.cs
using Xunit;

namespace None
{
    public class MyFluentWrapper
    {
        [Fact(Skip = "")]
        public void CallSiteShouldWorkForAsyncMethodsWithReturnValue()
        {
            {
                var callSite = GetAsyncCallSite().GetAwaiter().GetResult();
                Assert.Equal("NLog.UnitTests.LayoutRenderers.CallSiteTests.GetAsyncCallSite", callSite);
            }
        }
    }
}
